package.json

```diff
@@ -140,6 +143,7 @@
     "joi": "10.4.1",
     "jquery": "3.2.1",
     "js-yaml": "3.4.1",
+    "json-stringify-pretty-compact": "^1.0.4",
```

Comment:
Fixed version number.

Reply:
Do we really need another dependency, just so that we can have a max line width when reformatting? In my opinion this is a too less important feature, to pull in another library for.

Reply:
Hard to say - I use it daily for all the demos, as otherwise pure JSON is too long, but hjson lib includes it natively, so I guess we don't have to have both JSON & HJSON support it. It's a tiny but useful library.